neon_diana_utils/configuration.py

```diff
@@ -593,23 +645,48 @@ def configure_neon_core(mq_user: str = None,
         return
 
     # Prompt for IRIS Web UI configuration
-    confirmed = False
-    iris_domain = "iris.diana.k8s"  # TODO: Read from backend config
-    while not confirmed:
-        iris_domain = click.prompt("Hostname for Iris Web UI", type=str,
-                                   default=iris_domain)
-        confirmed = click.confirm(f"Is {iris_domain} correct?")
+    iris_domain = None
+    if click.confirm("Configure IRIS Gradio Web UI?"):
+        confirmed = False
+        iris_domain = "iris.diana.k8s"  # TODO: Read from backend config
+        while not confirmed:
+            iris_domain = click.prompt("Hostname for Iris Gradio Web UI",
+                                       type=str, default=iris_domain)
+            confirmed = click.confirm(f"Is {iris_domain} correct?")
 
```

Comment:
This logic doesn't quite work with confirmation - if you make a mistake, `iris_domain` becomes the previous mistaken value:

```sh
Hostname for Iris Gradio Web UI [iris.diana.k8s]: y
Is y correct? [y/N]: n
Hostname for Iris Gradio Web UI [y]: iris.diana.k8s
Is iris.diana.k8s correct? [y/N]: y
```

Reply:
Would it make more sense to set the default to `None` if not confirmed? I didn't really consider the default value behavior after an unconfirmed entry

Reply:
Probably easiest to just set a different variable from `iris_domain` on line 653, right? Reusing the variable is what's causing the issue, I think

Reply:
I guess if not confirmed then there's no reason to use the invalid thing as a default.. I'll just hard-code the default in the prompt